src/util/ui_message.h

```diff
@@ -94,10 +94,75 @@ class ui_message_handlert : public message_handlert
 
   std::string command(unsigned c) const override
   {
-    if(message_handler)
-      return message_handler->command(c);
-    else
+    if(!message_handler)
       return std::string();
+
+    switch(_ui)
+    {
+    case uit::PLAIN:
+      switch(c)
+      {
+      case '<': // fall through
+      case '>':
+        return "'";
+      }
+      break;
+    case uit::XML_UI:
+      switch(c)
+      {
+      case 0:  // reset
+      case 1:  // bold
+      case 2:  // faint
+      case 3:  // italic
+      case 4:  // underline
+      case 31: // red
+      case 32: // green
+      case 33: // yellow
+      case 34: // blue
+      case 35: // magenta
+      case 36: // cyan
+      case 91: // bright_red
+      case 92: // bright_green
+      case 93: // bright_yellow
+      case 94: // bright_blue
+      case 95: // bright_magenta
+      case 96: // bright_cyan
+        return std::string();
+      case '<': // quote_begin
+        return "<quote>";
```

Comment:
Not sure this is a good idea as it changes the XML output format. I'd stick to single quotes for now and introduce a different format in a separate PR if desired.

Reply:
I have now factored this into a commit of its own, which could thus easily be dropped from the series.

Reply:
I would really prefer to drop this commit. IMO, it causes unnecessary hassle to XML output users.

Reply:
Removed, just left behind a comment that this could be done.